protobuf/protoc-gen-govalidator/example/example.proto

```diff
@@ -66,6 +66,14 @@ message ExampleMessage {
     }
 
    string nested_email = 2 [(validator.regex) = ".+\\@.+\\..+"];
+
+   repeated string member_emails = 3 [
+      (validator.regex) =
+          "[a-z0-9!#$&'*+/=?^_{|}~-]+(?:\\.[a-z0-9!#$&'*+/=?^_{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?",
+      (validator.length_lte) = 200,
```

Comment:
This validator is not tested for in the new tests; should we? or should we just remove this validator from this example.

Reply:
that's true. makes sense to remove in this context.